tests/cli/test_main_notices.py

```diff
@@ -71,6 +73,43 @@ def test_main_notices(
             assert message not in captured.out
 
 
+def test_main_notices_json(
+    capsys,
+    conda_notices_args_n_parser,
+    notices_cache_dir,
+    notices_mock_fetch_get_session,
+    monkeypatch: MonkeyPatch,
+    mocker: MockerFixture,
+    status_code=200,
+):
+    channel_str = "test"
+    mocker.patch(
+        "conda.base.context.Context.channels",
+        new_callable=mocker.PropertyMock,
+        return_value=(channel_str,),
+    )
+    args, parser = conda_notices_args_n_parser
+    messages = ("Test One", "Test Two")
+    messages_json = get_test_notices(messages)
+    json_list = messages_json.get("notices")
+
+    for item in json_list:
+        item.update({"channel_name": channel_str, "interval": "null"})
```

Comment:
Would it be better to add this to `get_test_notices`? cc @travishathaway

Reply:
I think Travis didnt want to add this to the `get_test_notices` func itself because then we might need to change the other funcs where `get_test_notices` is being called.

Reply:
I agree that `channel_name` isn't part of the API endpoint but isn't `interval` included in `notices.json`? https://github.com/conda/conda/blob/078e7ee79381060217e1ec7f9b0e9cf80ecc8f3f/conda/notices/types.py#L56-L69

Reply:
@kenodegard,

No, it's not: https://github.com/conda-incubator/ceps/blob/main/cep-6.md#what-file-format-will-this-message-have-and-what-will-it-contain

I think I was originally planning to have it there and must have forgotten to remove it from conda 🙃.

Reply:
I could add removing (deprecating) it as an item for this issue:

- https://github.com/conda/conda/issues/13652

It would make sense to have that change there as we may wish to refactor some of that code anyways.